pages/scan-results/ScanResults.js

```diff
@@ -195,29 +197,44 @@ class ScanResults extends Component {
     this.setState({ packages })
   }
 
-  handleSortAlphabetic = () => {
-    const { packages } = this.state
-    const sortedList = packages.sort((packA, packB) =>
-      packA.name.localeCompare(packB.name)
-    )
+  setParamsAndState = ( sortMode ) => {
+    const updatedQuery = { ...Router.query, sortMode}
```

Comment:
I think you meant to destructure `this.props.router.query` here.
```suggestion
    const updatedQuery = { ...this.props.router.query, sortMode}
```